src/main/java/com/iota/iri/Iota.java

```diff
@@ -180,8 +181,13 @@ private TipSelector createTipSelector(TipSelConfig config) {
 
         TailFinder tailFinder = new TailFinderImpl(tangle);
         Walker walker = new WalkerAlpha(tailFinder, tangle, messageQ, new SecureRandom(), config);
-        return new TipSelectorImpl(tangle, ledgerValidator, entryPointSelector, ratingCalculator,
-                walker, milestoneTracker, config);
+        TipSelector tipSel = new TipSelectorImpl(tangle, ledgerValidator, entryPointSelector, ratingCalculator,
```

Comment:
If tipSelector equasl 'CONFLUX', this line is not need to run, in fact, it runs every time.